Added a file to repo

pankajpundir opened the pull request.

PeterBell committed:
Merge pull request #2 from githubteacher/master

Most Senseis should be respected

Commit:
Initial kill list

PeterBell committed:
Merge pull request #3 from deadlyvipers/kill_list

Initial kill list

Commit:
We dont need to kill Budd

pankajpundir committed:
Added a file to repo

codeschool-kiddo commented on introduction.md:
```diff
@@ -0,0 +1,2 @@
+This is a sample file.
```
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

pankajpundir committed:
Addressed notes for favorite course path on code school.

pankajpundir committed:
Added my deadly skills

pankajpundir committed:
Added the requested skill

codeschool-kiddo committed:
Added new rule about coffee

pankajpundir committed:
Fixed the typos.

pankajpundir committed:
Only doing the requested changes. Nothing extra.

pankajpundir committed:
Merge pull request #2 from pankajpundir/new_rules

Too many people are spilling coffee

pankajpundir committed:
Merge branch 'deadly_skills'

pankajpundir committed:
Merge https://github.com/pankajpundir/dojo_rules

pankajpundir committed:
hot fix for v1.0.0

pankajpundir committed:
Merge branch 'r1.0.0'

pankajpundir committed:
going after bad coding practices

pankajpundir committed:
hot fix from release branch 1.0 with tag v1.0.1

pankajpundir committed:
Added my personal grievance, fixes #3

pankajpundir committed:
Merge pull request #4 from pankajpundir/kill_list

Added my personal grievance, fixes #3

pankajpundir committed:
This s how you write a ReadMe.md

pankajpundir committed:
Not so deadly_vipers need that line.

pankajpundir committed:
codeschool, update the test cases

Please update your test cases as my submission failed when I converted the text rules into a link. Hope it goes through this time.
